lsb_vsx/build.sh

```diff
@@ -13,6 +13,9 @@ PKG_MIRROR_URL2="https://files.phoesys.com/ports/${LSB_VSX}/lts_vsx-pcts2.0beta.
 PREFIX_LSB_VSX_MARKERS="${PREFIX_PORT_BUILD}/markers"
 PREFIX_LSB_VSX_FILES="${PREFIX_PORT_BUILD}/files"
 
+# Disable parallel build
+export MAKEFLAGS="${MAKEFLAGS}${MAKEFLAGS:+ }-j1"
```

Comment:
we could probably skip `${MAKEFLAGS:+ }` and always use ` ` (space) here.

Maybe add info in comment regarding WHY you're doing the change - (eg. "lsb_vsx code/build system is old and not prepared for concurrent build").

Maybe remove the patches regarding the `.NOTPARALLEL` build?

Reply:
Removed those patches, comment added. I think `${MAKEFLAGS:+ }` should stay to avoid potential misinterpretation.